src/main/java/calculator/Calculator.java

```diff
@@ -0,0 +1,71 @@
+package calculator;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Calculator {
+
+    public static final String CUSTOM_REGEX = "//(.)\n(.*)";
+    public static final String REGEX = ",|:";
+
+    public static int splitAndSum(String text) {
+        if (isBlank(text)) {
+            return 0;
+        }
+
+        String[] texts = split(text);
+        int[] ints = toNumbers(texts);
+        return sum(ints);
+    }
+
+    private static boolean isBlank(String text) {
+        return text == null || text.isBlank();
+    }
+
+    private static String[] split(String text) {
+        Matcher matcher = Pattern.compile(CUSTOM_REGEX).matcher(text);
+        if (matcher.find()) {
+            String customDelimiter = matcher.group(1);
+            return matcher.group(2).split(customDelimiter);
+        }
+
+        String[] result = text.split(REGEX);
+        return result;
+    }
+
+    private static int[] toNumbers(String[] texts) {
+        int[] numbers = new int[texts.length];
+        for (int i = 0; i < texts.length; i++) {
+            numbers[i] = parse(texts[i]);
+        }
+
+        return numbers;
+    }
+
+    private static int parse(String text) {
+        int number = Integer.parseInt(text);
+        if (checkNegative(number)) {
+            throw new RuntimeException();
+        }
+
+        return number;
+    }
+
+    private static boolean checkNegative(int number) {
+        if (number < 0) {
+            return true;
+        }
+
+        return false;
```

Comment:
```suggestion
       return number < 0;
```

간단한 로직은 간단하게 표현하면 좋을것 같아요!

Reply:
저렇게 간단하게 표현을 해도 되겠네요!!